src/tests/views/widgets/image.test.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import test from 'ava';
+import { shallow } from 'enzyme';
+import Image from '../../../ts/views/widgets/image';
+
+test('Image gets classnames properly and img recieves ', (t) => {
+  const wrapper = shallow(<Image url={`http://via.placeholder.com/350x150`} className="test-image"/>);
+  t.is(wrapper.hasClass('image-component test-image'), true);
+});
```

Comment:
write another test where you don't pass in a `url` and assert that resulting `style` is as expected (aka no `backgroundImage` style is set)